packages/patternfly-4/react-core/src/components/Select/SelectOption.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Select/select.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { SelectContext } from './selectConstants';
+
+const propTypes = {
+  /** the value for the option */
+  children: PropTypes.string,
+  /** additional classes added to the Select Option */
+  className: PropTypes.string,
+  /** the value for the option */
+  value: PropTypes.string,
+  /** flag indicating if the option is disabled */
+  isDisabled: PropTypes.bool,
+  /** Optional on click callback */
+  onClick: PropTypes.func,
+  /** Callback for ref tracking */
+  sendRef: PropTypes.func,
+  /** Additional props are spread to the container <button> */
+  '': PropTypes.any
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  value: null,
+  isDisabled: false,
+  onClick: Function.prototype
+};
+
+class SelectOption extends React.Component {
+  ref = React.createRef();
+
+  componentDidMount() {
+    this.props.sendRef(this.ref, this.props.index);
+  }
+
+  onKeyDown = event => {
+    if (event.key === 'Tab') return;
+    event.preventDefault();
+    if (event.key === 'ArrowUp') {
+      this.props.keyHandler(this.props.index, 'up');
+    } else if (event.key === 'ArrowDown') {
+      this.props.keyHandler(this.props.index, 'down');
+    } else if (event.key === 'Enter') {
+      this.ref.current.click && this.ref.current.click();
+    }
+  };
+
+  render() {
+    const {
+      children,
+      className,
+      value,
+      onClick,
+      isDisabled,
+      selected,
+      sendRef,
+      keyHandler,
+      index,
+      ...props
+    } = this.props;
+    return (
+      <SelectContext.Consumer>
+        {onSelect => (
+          <li>
```

Comment:
missing aria-pressed here.

Reply:
What is the usage of aria-pressed? Is it set to true or a value, and is it only when the li is selected?

Reply:
@jgiardino can you give some feedback on this please?

Reply:
I commented on this below, but `aria-pressed` was an earlier implementation based on not managing focus, and was used as a way to indicate current selection.